src/components/product/product.service.ts

```diff
@@ -368,10 +324,18 @@ export class ProductService {
       mediums: true,
       purposes: true,
       methodology: true,
+      scriptureReferences: true,
+      scriptureReferencesOverride: true,
+      produces: true,
     });
-    const baseNodeProps = parseBaseNodeProperties(result.node);
 
-    const produces = await this.db
+    const {
+      produces,
+      scriptureReferencesOverride,
+      ...rest
+    } = securedProperties;
```

Comment:
This destructure could be merged with the assignment above.
`props` is also not being used so `result.propList` could just be passed as the first arg to `parseSecuredProperties`.